src/components/AddItem.js

```diff
@@ -46,6 +64,7 @@ const AddItem = ({ token }) => {
             value={formData.itemName}
             onChange={handleFormChange}
           />
+          {hidden ? <Duplicate /> : ''}
```

Comment:
Would this be the same as `{hidden ? <Duplicate /> : null}` ? I've never seen it done with an empty string, just wondering if there is an advantage to one over the other.

Reply:
The absence of a component using `null` makes much more sense than having an empty string. Thanks for pointing this out!

Reply:
I did some research and it looks like they are mostly interchangeable. They are both read as `falsy` by the JSX. However, I seem to remember hearing that there may be some unique cases where one might be used over the other, but I can't track down where I heard / read that.